src/internal/models.py

```diff
@@ -314,3 +314,24 @@ class Meta:
 
     def __str__(self):
         return _("“{quote}” —{quoted}").format(quote=self.quote, quoted=self.quoted)
+
+
+class Lore(models.Model):
+    title = models.CharField(max_length=200, unique=True, verbose_name=_("title"))
+    slug = models.SlugField(unique=True)
+    content = RichTextUploadingField(max_length=150000, verbose_name=_("text"))
+
+    def __str__(self):
+        return self.title
+
+    def save(self, *args, **kwargs):
+        title = self.title
+        self.title = capfirst(title)
+        title = title.replace('ø', 'o')
+        title = title.replace('æ', 'ae')
+        self.slug = slugify(title)
+        super().save(*args, **kwargs)
+
+    class Meta:
```

Comment:
Ah btw, the `Meta` class should be placed before `__str__()`, as per [the code style guide](https://github.com/MAKENTNU/web/blob/main/CONTRIBUTING.md#field-and-method-order) :)